fnf/README.md

```diff
@@ -32,3 +32,21 @@ test_postgresdb = postgres+psycopg2://postgres@localhost/postgres
 
 `disinfo`: the project's database.  
 `postgres`: default database that is shipped with PostgreSQL and used here for testing the ORMs.
+
+## How to collect sample data from MAG to run `parse_mag.py` ##
+Instead of getting all the data for a MAG Field of Study by running the [`query_fos_mag.py`](https://github.com/nestauk/fnf/blob/dev/fnf/query_fos_mag.py), it's preferable to get a small subset in order to test that the MAG response is loaded, parsed and stored in `postgres` correctly.
+
+You should do the following:
+
+``` python
+python sample_mag_data.py
```

Comment:
:+1: This helps for reviewing

We should probably think about  adding in test modes for cases like this, but we would need to think about the cleanest way to do it.